{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -71,3 +71,5 @@ ENTRYPOINT ["/sbin/tini", "--", "/docker-entrypoint.sh"]
 FROM development_build as production_build
 
 COPY . /code
+
+RUN python /code/manage.py compilemessages
```

Comment:
This way we won't have compiled messages in development. What do you think?

Reply:
We currently don't have them as well

Reply:
Why we should have them in development?

Besides, in development, we mount code, not copy it. If a developer needs something to do with messages, he will mount them and compile manually, isn't it?

Reply:
Good enough!